cli/src/pcluster/templates/cluster_stack.py

```diff
@@ -127,6 +136,39 @@ def _get_compute_security_groups(self):
 
         return compute_group_set
 
+    # -- Parameters -------------------------------------------------------------------------------------------------- #
+
+    def _add_parameters(self):
+        CfnParameter(
+            scope=self,
+            id="ClusterUser",
+            description="Username to login to head node",
+            default=OS_MAPPING[self.config.image.os]["user"],
+        )
+        CfnParameter(
+            scope=self,
+            id="ResourcesS3Bucket",
+            description="S3 user bucket where AWS ParallelCluster resources are stored",
+            default=self.bucket.name,
+        )
+
+        CfnParameter(
+            scope=self,
+            id="ArtifactS3RootDirectory",
+            description="Root directory in S3 bucket where cluster artifacts are stored",
+            default=self.bucket.artifact_directory,
+        )
+
+        CfnParameter(id="Scheduler", scope=self, default=self.config.scheduling.scheduler)
+
+        if self.config.is_cw_logging_enabled:
+            CfnParameter(
+                scope=self,
+                id="ClusterCWLogGroup",
```

Comment:
Do we need it as parameter? I mean do we need it for failed stacks? If so it's good to keep them as input.